messaging/src/test/java/org/axonframework/eventhandling/MultiSourceTrackingTokenTest.java

```diff
@@ -136,6 +136,12 @@ public void covers() {
         newTokens.put("token2", new GlobalSequenceTrackingToken(0));
 
         assertTrue(testSubject.covers(new MultiSourceTrackingToken(newTokens)));
+
+        newTokens.put("token2", null);
```

Comment:
To be honest, I'd prefer a separate named test class for this null case you've uncovered here @sgrimm-sg.
Would you mind moving the asserts in the `covers` and `doesNotCover` into a different method?

Reply:
I've moved these to a new method.